General/Baselines.md

```diff
@@ -2,6 +2,11 @@
 
 Pharo projects often requires a configuration to declare the how they should be loaded. This configuration is done via **Baselines**. A baseline defines the packages of the project, their dependencies to each other and to external projects and independant sub-groups that can be loaded.
 
+Adding a baseline to a project has some advantages:
+- It makes it easier to load the project and for others to contribute to your project
+- It allows the users of the project to be unaware of the project's dependencies
```

Comment:
For me it allows to be aware of the dependencies. Because you can see the dependencies declarations in the Baseline.

But you point is to say that user do not need to load dependencies by hand I guess?

Reply:
Yes!

I'll add another point with what you said